web/main/templates/export/as_printable_html/node.html

```diff
@@ -22,13 +22,13 @@ <h2 class="subtitle">{{ node.subtitle|default_if_none:"" }}</h2>
         <section class="headnote">{{ headnote }}</section>
       {% endif %}
 
-      <a href="{{ node.resource.url }}" target="_blank" data-type="resource">{{ node.resource.url }}</a>
+      <a href="{{ node.resource.url }}" target="_blank" data-type="resource link">{{ node.resource.url }}</a>
 
     {% else %}
 
         <h1 class="{{ node.type }} title">
           <span class="{{ node.type }} ordinal" data-toc-idx="{{ index }}">{{ node.ordinal_string }}</span>
-          {{ node.title }}
+          <span>{{ node.title }}</span>
```

Comment:
Not blocking at all but for my curiosity: what's the value of adding a span here?

Reply:
So that it's addressable by this rule:

https://github.com/harvard-lil/h2o/pull/1790/files#diff-ecb321e58f1547e83335bbc1669dfcf87b70a66e423d70a2ac09b2dca4ec888aR36

Otherwise it would inherit the default span behavior of being an inline. Off the top of my head I don't remember how it flowed differently as an inline though!